### DISCUSSION TITLE
Add an arrow to a plugin through tools which expands to other options on hover

### DISCUSSION THREAD

**Discussion by deqline** [Vector35/binaryninja-api#1736] [General] [answered]
So I would like to be able to achieve that for my plugin (the arrow which then expands to multiple options): 
![](https://i.imgur.com/JeBgfyo.png)

Also how would I implement an overlay when clicking on one of these expanded functions?

Thanks in advance.

**Comment by psifertex** [comment, marked as answer]
Checkout the "name" parameter docs: https://api.binary.ninja/binaryninja.plugin-module.html#binaryninja.plugin.PluginCommand.register

TL;DR put "Main\\Sub" for the name. (I think the docs should have a double backslash, but not positive)

**Comment by psifertex** [comment]
Correct. Just call register multiple times with each callback function.
